common/templates/xblock_v2/xblock_iframe.html

```diff
@@ -437,21 +437,55 @@
         const rootNode = document.querySelector('.xblock, .xblock-v1'); // will always return the first matching element
         initializeXBlockAndChildren(rootNode, () => {
         });
+        (function() {
+            // If this view is rendered in an iframe within the authoring microfrontend app
+            // it will report the height of its contents to the parent window when the
+            // document loads, window resizes, or DOM mutates.
+            if (window !== window.parent) {
+                var lastHeight = window.parent[0].offsetHeight;
+                var lastWidth = window.parent[0].offsetWidth;
```

Comment:
@navinkarkera Apparently these lines are causing same-origin errors:

```
Uncaught SecurityError: Failed to read a named property 'offsetHeight' from 'Window': Blocked a frame with origin "https://studio.stage.edx.org/" from accessing a cross-origin frame.
```

Can we change the initial value of `lastHeight` and `lastWidth` to something that doesn't try to access `window.parent`?

Reply:
@bradenmacdonald Turns out that we don't even need this piece of code.

Created a very small PR: https://github.com/openedx/edx-platform/pull/36643